src/python/dxpy/scripts/dx_build_app.py

```diff
@@ -36,6 +36,10 @@
 import dxpy.workflow_builder
 import dxpy.executable_builder
 from .. import logger
+from pathlib import Path
```

Comment:
`pathlib` is not available in Python 2.7, it was introduced in 3.4. I don't see this module being used anywhere so can it be safely removed?

Reply:
Thanks! Removed